x-pack/plugins/cases/public/components/create/template.test.tsx

```diff
@@ -53,7 +53,63 @@ describe.skip('CustomFields', () => {
     );
 
     await waitFor(() => {
-      expect(onTemplateChange).toHaveBeenCalledWith(selectedTemplate.caseFields);
+      expect(onTemplateChange).toHaveBeenCalledWith({
```

Comment:
Can you please add a test that confirms `onTemplateChange` is not called when no `templateToSelect` is passed?

Reply:
[added check](https://github.com/elastic/kibana/pull/190701/commits/c55a653aa32136db27b898aa532cfa3f8f37eb9b#diff-5c4e6ae25f523f9068fc9206433db412001735d35852d9d22aada2ca98e74fef)